modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Headless Commerce Admin Catalog API
 Bundle-SymbolicName: com.liferay.headless.commerce.admin.catalog.api
-Bundle-Version: 36.3.1
+Bundle-Version: 37.0.0
```

Comment:
not sure, we need to avoid breaking changes on APIs, can we find another way to implement this?